model.md

```diff
@@ -5,19 +5,26 @@ Consider the basic scenario of a school looking for a list of suitable and avail
 
 ## The teacher profile
 We can identify the following minimum properties
-* **Essential personal data**
-    * unique identifier, typically an email address or a chose username
-    * First name and surname
-    * Email
-    * Place of residence, possibly only postal code. This is necessary to make a geographical match.
-    * Country. At present littil is only active in the Netherlands, but this can change
-    * Non-essential personal data would be date of birth, gender, telephone, full postal address. It is debatable whether we need at all. The less, the better
-    * Nice to have and without obvious privacy constraints would be a link to a public profile, like a personal website or LinkedIn. A few lines of bio would also be good. 
-* **Teaching skills, preferences and availability**
-  * How experienced is the teacher in giving classes in general? Have they given similar classes before?
-  * What is their area of expertise, with regard to the choice of classes they might provide?
-  * What weekdays are they available?
-  * How far are they willing to travel?
+* unique identifier, generated by the platform. Preferably a UUID, not a DB sequence number.
```

Comment:
Do we need to think about the version of the UUID? From what I've read Mysql support version 1 out of the box. Default of java (randomUUID()) is version 4. Anyone experience with Mysql and UUID?

Reply:
Is this an extra identifier apart from a primary key field? Don't know why it explicitly needs to not be a DB sequence number, but if you want this UUID to be in the URL or something a v1 might be random enough.

Reply:
Not being tied to the DB's mechanism of auto-key generation makes it more portable, I think.

Reply:
True, then we should also generate UUIDs in code I guess.